survey/performAnalysis.sh

```diff
@@ -0,0 +1,26 @@
+# pip install -r requirements.txt
+cd scripts
+python combineSurveyData.py \
+  ../raw_data/fake_data_1.csv \
+  ../raw_data/fake_data_max-diff_1.csv \
+  ../raw_data/fake_data_2.csv \
+  ../raw_data/fake_data_max-diff_2.csv
```

Comment:
I suspect we don't want "fake_data" showing up here?

Reply:
IMO it's useful to have because the filenames matter and it shows that there needed to be two files for each survey branch (max-diff and regular) and shows how the script was called

Reply:
Yes but people won't edit this file - let's be honest. They are going to assume that if they copy the data and put it in "raw_data", it'll just work.

Reply:
Would you prefer it have the real filenames?

Reply:
Yup, as they are listed here: https://zenodo.org/records/11150653